File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.py
class Solution(object):
    def check(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        n = len(nums)
        count=0
        if(nums[0]<nums[n-1]):
            count=count+1
        for i in range(1,n):
               if(nums[i]<nums[i-1]):
                    count=count+1    
        return False if count>1 else True

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        
        int count=0;
        int  n=nums.size();
        if(nums[0]<nums[n-1]) count++;
        for(int i=1;i<n;i++){

            if(nums[i-1]>nums[i])
                count++;
        }

        return count>1? false:true;
    }
};